content/about/history.md

```diff
@@ -19,16 +19,16 @@ layout: "general"
 <div class="panel-body mb-4">
 <div class="row">
 <div class="col-lg-8 col-sm-12">
-<p>GRASS GIS has been under continuous development since <b>1982</b>, when the U.S. Army Corps of Engineers' Construction Engineering Research Laboratory (USA/CERL)
+<p>GRASS (Geographic Resources Analysis Support System) has been under continuous development since <b>1982</b>, when the U.S. Army Corps of Engineers' Construction Engineering Research Laboratory (USA/CERL)
 started exploring the use of GIS for environmental research, monitoring and management of military lands.</p>
-<p>Since no other software package available back then met all their requirements, they designed and  developed their own. GRASS GIS was born under the name of Fort Hood Information System (FHIS). Meanwhile, other US federal agencies and environmental offices became interested and the more general purpose <b>GRASS was born</b>. </p>
+<p>Since no other software package available back then met all their requirements, they designed and  developed their own. GRASS was born under the name of Fort Hood Information System (FHIS). Meanwhile, other US federal agencies and environmental offices became interested and the more general purpose <b>GRASS was born</b>. </p>
```

Comment:
```suggestion
<p>Since no other software package available back then met all their requirements, they designed and developed their own. GRASS was born under the name of Fort Hood Information System (FHIS). Meanwhile, other US federal agencies and environmental offices became interested, and the more general purpose <b>GRASS was born</b>. </p>
```